### ISSUE TITLE
Update Bounty awarded event

### ISSUE THREAD

**Issue by Darkcyde13** [EDCD/EDDI#2506]
## What happens now
Currently, when you scan a civilian after killing them, the `Bounty awarded` event triggers and they are all called `Worker` for `event.target`, no matter what type they are.

## What I'd like to happen 
I'd like the correct identification to be used, like `Researcher` or `Technician`.

## How it can happen
The Journal details the correct target type when the corpse is scanned.  This information should be used instead of the generic `Worker` (although that could still be used as an option).
```
{ "timestamp":"2023-01-28T23:36:38Z", "event":"Bounty", "Rewards":[ { "Faction":"Duwali Liberty Party", "Reward":2050 } ], "Target":"citizensuitai_scientific", "Target_Localised":"Researcher", "TotalReward":2050, "VictimFaction":"Duwali Partnership" }
{ "timestamp":"2023-01-28T23:36:59Z", "event":"Bounty", "Rewards":[ { "Faction":"Defence Party of Duwali", "Reward":14100 } ], "Target":"citizensuitai_industrial", "Target_Localised":"Technician", "TotalReward":14100, "VictimFaction":"Duwali Partnership" }
```
## EDDI Version
v4.0.2

**Comment by Tkael** [comment]
Hmm. According to my tests they should have been called "Scientist" and "Worker". 
I agree that it would be better to use the same localizations as the game, e.g. "Researcher" and "Technician".

**Comment by Tkael** [comment]
Resolved by b465a22adac5178a447c9c73002997286e61ba4c.
